_config/log.yml

```diff
@@ -18,7 +18,7 @@ SilverStripe\Core\Injector\Injector:
     class: Monolog\Handler\StreamHandler
     constructor:
       - '../log/csp.log'
-      - '`LOG_LEVEL`'
+      - 200
     properties:
```

Comment:
Some weirdness going on here, the LOG_LEVEL env doesn't seem to get past into the constructor, causing an error. Hard coding it 'fixes' the issue.

Reply:
got anything in your .env file for log_level?